### PR TITLE
Add option to directly copy the password to clipboard

### PR DESCRIPTION
**Pull request by vrunoa**

### CODE REVIEW

File: wifi-password.sh [hunk 1]
```diff
@@ -94,6 +102,9 @@ fi
 if [ $verbose ]; then
   echo "\033[96m ✓ \"$pwd\" \033[39m"
   echo ""
+elif [ $copy ]; then
+  echo $pwd | pbcopy
+  echo "\033[96m ✓ Password copied to clipboard"
 else
   echo $pwd
-fi
+fi
```
Comment (albinekb) [inline wifi-password.sh]: Newline!

Comment (vrunoa) [inline wifi-password.sh]: Sorry @albinekb I didnt not understand. Should I add or remove a newline

Comment (albinekb) [inline wifi-password.sh]: image
http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file
:)

Comment (vrunoa) [inline wifi-password.sh]: I think it's fixed 😄

**Pull request closed without merging**